projects/01_Google/Özgür/style.css

```diff
@@ -0,0 +1,243 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.3rem;
+  font-family: arial, sans-serif;
+}
+
+.nav-container {
+  display: flex;
+  margin: 0.5rem;
+}
+
+.nav-items-left,
+.nav-items-right {
+  display: flex;
+  margin: 5px;
+}
+
+.nav-items-right {
+  margin-left: auto;
+}
+
+.nav-container .list-group-items {
```

Comment:
After making the necessary changes, it will be sufficient to write it as follows.
nav ul li {
  list-style-type: none;
  padding: 1em;
}